src/fileio/impls/hdf5/h5_format.cxx

```diff
@@ -666,7 +671,7 @@ bool H5Format::read_rec(void *data, hid_t hdf5_type, const char *name, int lx, i
 }
 
 bool H5Format::write_rec(int *data, const char *name, int lx, int ly, int lz) {
-  return write_rec(data, H5T_NATIVE_INT, H5T_NATIVE_INT, name, lx, ly, lz);
```

Comment:
Why has the number of arguments changed?

Reply:
`write_hdf5_type` is used to initially create the variable, but that functionality has now been moved into `H5Format::addVar` so `write_hdf5_type` is no longer used here.

This is a private method so doesn't change the public API